common/dependabot-common.gemspec

```diff
@@ -35,15 +35,11 @@ Gem::Specification.new do |spec|
   spec.add_dependency "parser", ">= 2.5", "< 4.0"
   spec.add_dependency "toml-rb", ">= 1.1.2", "< 3.0"
 
-  spec.add_development_dependency "debase", "0.2.3"
-  spec.add_development_dependency "debase-ruby_core_source", "0.10.16"
-  spec.add_development_dependency "debug", ">= 1.0.0"
```

Comment:
This pin is removed because it's now part of stdlib as of ruby 3.1.

Reply:
Weirdly, the tests are now failing complaining about unable to find `"debug"`, but I verified it's using Ruby 3.1.2, so not sure why it can't find `"debug"`...

Reply:
Hei @jeffwidman!

Stdlib (default) gems don't need to be explicitly added as dependencies, although in my opinion, it's a good practice to add them anyways. But they do need to be explicitly required.

In this case, though, `debug` is not a "default gem", but a "bundled gem". The distinction is subtle but matters here. You can find more about it at https://stdgems.org.

While in the case of default gems you can always use them because they are "part of Ruby", in the case of bundled gems, that depends on your gem setup. In our case, we use a customized `GEM_PATH` (the path to install and look for gems), so the `debug` gem bundled with Ruby by default is no longer found since it lives in the default `GEM_PATH`.

So, in conclusion, I believe we need to keep the pin.

Reply:
Thank you @deivid-rodriguez that's very helpful.

Reply:
I just put back the pin and updated the relevant commit message, so it should pass CI now🤞.